### PR TITLE
starting to add contributing guidelines

### PR DESCRIPTION
**Pull request by Jayme-T**

### CODE REVIEW

**Comment by Jayme-T** [comment]
added contributing.md

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+People are making and building Private Elements for Cloud Elements at rapid rates. To contribute we ask that you create a folder that contains the Element json as well as a short readme file with a description of what you built (endpoint name and resources available so people can see them without importing the Element), a link to the documentation you used when creating the Element, and any known endpoint limitation that might affect how your Element works.
```
Comment (rrothenb) [inline CONTRIBUTING.md]: Since you're being really good about capitalization, it probably makes sense to also capitalize JSON

Comment (rrothenb) [reply CONTRIBUTING.md]: I feel like maybe README should be capitalized, though I'm not sure.

Comment (rrothenb) [reply CONTRIBUTING.md]: Very nice.

Comment (Jayme-T) [reply CONTRIBUTING.md]: Oh good idea! Thanks

**Pull request merged**